Forms/Main.cs

```diff
@@ -1405,6 +1406,8 @@ private ListViewItem DownloadListItem(Model.Download info, ListViewItem item)
                         Model.Programme progInfo = new Model.Programme(info.Progid);
                         item.SubItems[column].Text = progInfo.Name;
                         break;
+                    case Model.Download.DownloadCols.SmartName:
```

Comment:
I'm not sure why we aren't setting the smart name here?

Reply:
Do you mean we could be adding the 'SmartName' value here rather than in `private void InitDownloadList()` function (line 2495)? I'd certainly be interested to understand how; vaguely recall looking a this early on but didn't understand how i'd do that.

Reply:
Yes, it should just be something along the lines of
``` c#
Model.Programme progInfo = new Model.Programme(info.Progid);
item.SubItems[column].Text = TextUtils.StripProgrammeNameFromEpisode(progInfo.Name, info.Name);
```